src/GitHub/Request.hs

```diff
@@ -256,8 +256,8 @@ makeHttpRequest auth r = case r of
     setAuthRequest _                                  = id
 
     getOAuthHeader :: Auth -> RequestHeaders
-    getOAuthHeader (OAuth token) = [("Authorization", BS8.pack ("token " ++ token))]
-    getOAuthHeader _                   = []
+    getOAuthHeader (OAuth token) = [("Authorization", "token " <> token)]
+    getOAuthHeader _             = []
```

Comment:
It [seems](https://github.com/phadej/github/search?utf8=✓&q=EnterpriseOAuth) that `EnterpiseOAuth` is only used to provide an endpoint.

Admittedly, I know nothing about enterprise installations.

Also, according to [GitHub's docs](https://help.github.com/enterprise/2.4/admin/guides/user-management/authenticating-users-for-your-github-enterprise-instance/)

> Note: User authentication using GitHub OAuth is deprecated and can not be used for new installations.